nodejs/nodejs-guestbook/skaffold.yaml

```diff
@@ -1,5 +1,4 @@
-
-apiVersion: skaffold/v1beta15
+apiVersion: skaffold/v2beta5
```

Comment:
I don’t think we can update the apiVersion yet. Our LSP does not yet support it. We plan to do this in bulk (@michihara has a PR staged for this already) for the next release where we expose buildpack support

Reply:
https://github.com/GoogleCloudPlatform/cloud-code-samples/pull/355

Reply:
Could we separate out this part from this fix?

Reply:
'k, will back that out.

Reply:
thanks. lgtm. will approve, but I think @simonz130 needs final approval / will merge

Reply:
Backed out to v2beta4; v1beta15 doesn't support the `buildpack` builder.